tensorboard/tools/build_lint.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+ret_val=0
+
+for build_path in "$@"
+do
+  # We do not care about few lint warns just yet.
+  buildifier --mode=check --lint warn "$build_path" 2>&1 | grep -v -E "(load-on-top:|native-py:|native-java:|constant-glob:)" 1>&2
```

Comment:
By passing each file separately to `buildifier` and `grep`, you’re
spawning 190 times as many processes as you would otherwise, and forcing
`buildifier` to reinitialize each time. This makes the script take 3.6
seconds instead of 0.136 seconds on my machine.

From the code inside the loop, it looks we could probably use [the
`--warnings` argument to `buildifier`][warn] to process these all at
once. (We wouldn’t get the fancy return code, but I think that that’s
okay.) What do you think?

[warn]: https://github.com/bazelbuild/buildtools/blob/master/buildifier/README.md#linter

Reply:
The warnings flag is perfect and I really think it is useful. 

> We wouldn’t get the fancy return code,

There are problems with the code I wrote as you mentioned below and buildifier also does have pretty cool return code so I am happy to use vanilla buildifier :)

```
Return codes used by buildifier:

  0: success, everything went well
  1: syntax errors in input
  2: usage errors: invoked incorrectly
  3: unexpected runtime errors: file I/O problems or internal bugs
  4: check mode failed (reformat is needed)
```